content/quickstart/ios.md

```diff
@@ -104,6 +111,31 @@ Practically the codes should not be structured this way. It is for demo only.
     }];
 
 ```
+```swift
+// Every record in Skygear must be owned by a user
+// For testing purpose, we have used signupAnonmously to create a record
+// Visit the user authetication documentation to learn more
+// https://docs.skygear.io/guides/auth/basics/ios/
+
+let skygear = SKYContainer.default()
+skygear?.signupAnonymously(completionHandler: { (user, error) in
+    if error != nil {
+        print("Signup Error: \(error?.localizedDescription)")
+        return
+    }
+    
+    // Create the table "test" and the record "Hello world"
```

Comment:
I think we can address it more clearly. Using both Skygear's term (like Record type) and explain what's happening behind the scene.